loader/test/albertsons.test.js

```diff
@@ -206,6 +212,83 @@ describe("Albertsons", () => {
     expect(result[0]).toHaveProperty("availability.products", ["pfizer"]);
   });
 
+  it("handles separate adult and pediatric entries for the same location", async () => {
+    nock(API_URL_BASE)
+      .get(API_URL_PATH)
+      .query(true)
+      .reply(
+        200,
+        [
+          {
+            id: "1635993536219",
+            region: "Eastern_-_6",
+            address:
+              "Pfizer Child - Safeway 0005 - 11120 South Lakes Drive, Reston, VA, 20191",
+            lat: "38.939784",
+            long: "-77.332298",
+            coach_url: "https://kordinator.mhealthcoach.net/vcl/1635993536219",
+            availability: "yes",
+            drugName: ["PfizerChild"],
+          },
+          {
+            id: "1600100807144",
+            region: "Virginia",
+            address:
+              "Safeway 0005 - 11120 South Lakes Drive, Reston, VA, 20191",
+            lat: "38.939784",
+            long: "-77.332298",
+            coach_url: "https://kordinator.mhealthcoach.net/vcl/1600100807144",
+            availability: "yes",
+            drugName: ["Pfizer", "Moderna", "JnJ"],
+          },
+        ],
+        { "Last-Modified": "Thu, 28 Oct 2021 07:06:13 GMT" }
+      );
+
+    const result = await checkAvailability(() => {}, { states: "VA" });
+    expect(result).toContainItemsMatchingSchema(locationSchema);
+    expect(result).toEqual([
+      {
+        name: "Safeway 0005",
+        external_ids: [
+          ["albertsons", "1635993536219"],
+          ["albertsons_safeway", "1635993536219"],
+          ["safeway", "0005"],
+          ["albertsons_store_number", "safeway:0005"],
+          ["albertsons", "1600100807144"],
+          ["albertsons_safeway", "1600100807144"],
+        ],
+        location_type: "PHARMACY",
+        provider: "albertsons",
+        address_lines: ["11120 South Lakes Drive"],
+        city: "Reston",
+        state: "VA",
+        postal_code: "20191",
+        position: {
+          longitude: -77.332298,
+          latitude: 38.939784,
+        },
+        info_url: "https://www.safeway.com/pharmacy/covid-19.html",
+        booking_url: "https://kordinator.mhealthcoach.net/vcl/1600100807144",
+        availability: {
+          source: "univaf-albertsons",
+          available: "YES",
+          products: ["pfizer_age_5_11", "pfizer", "moderna", "jj"],
+          is_public: true,
+          checked_at: expectDatetimeString(),
+          valid_at: "2021-10-28T07:06:13.000Z",
+        },
+        meta: {
+          albertsons_region: "Virginia",
+          booking_url_adult:
+            "https://kordinator.mhealthcoach.net/vcl/1600100807144",
+          booking_url_pediatric:
+            "https://kordinator.mhealthcoach.net/vcl/1635993536219",
+        },
```

Comment:
I couldn’t think of a better approach to preserving this info than putting it in `meta`. Will have to notify Anchorage and NJ folks about it.

I wanted to keep the top-level `booking_url` field as a nice, deep link, but it now has the downside of only working for adult vaccines. Another option is to put a more general link like https://www.mhealthappointments.com/covidappt in the top level, so it works for all vaccine types.